src/client.ts

```diff
@@ -71,8 +71,8 @@ function validateOptions (options: ISdkConfig): string | null {
  * SDK to interact with GenesysCloud WebRTC functionality
  */
 export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEventEmitter<EventEmitter, SdkEvents> }) {
-  readonly VERSION = '__GENESYS_CLOUD_WEBRTC_SDK_VERSION__';
-  logger: ILogger;
+  static readonly VERSION = '__GENESYS_CLOUD_WEBRTC_SDK_VERSION__';
+  logger: Logger;
```

Comment:
This isn't really a breaking change, although I guess it _could_ mess with some typings potentially.